Templates/WebHook-Generic-CSharp/run.csx

```diff
@@ -0,0 +1,25 @@
+﻿#r "Newtonsoft.Json"
+
+using System;
+using System.Net;
+using System.Net.Http;
+using Microsoft.Azure.WebJobs.Host;
+using Newtonsoft.Json;
+
+public static object Run(HttpRequestMessage req, TraceWriter log)
+{
+    log.Verbose($"Webhook was triggered!");
+
+    string jsonContent = req.Content.ReadAsStringAsync().Result;
```

Comment:
Note that `#r "System.Net.Http.Formatting"` does not work, probably because it's not in your list. We should just add it by default. Opened https://github.com/Azure/azure-webjobs-sdk-script/issues/91.